src/components/StockSearchBar/ShowStock/index.jsx

```diff
@@ -0,0 +1,43 @@
+import { useColorModeValue } from '@chakra-ui/react';
+import React from 'react';
+import styled from 'styled-components';
+
+const StockContainer = styled.div`
```

Comment:
i think we are not use html with css 
Can we please use HTML with css for consistency 
+ we are not using styled components in the project either

Reply:
Sure, will refactor.